### ISSUE TITLE
Launch window/inclination not correct when launching to the same plane as moon from KSC

### ISSUE THREAD

**Issue by jenden0** [TheBassist95/Kos-Stuff#1]
First off, awesome scripts.  I had played around with some simple launch scripts before doing RO but it was all much easier when you didn't have to worry about throttle ranges and engine ignition counts.
I saw the post on reddit but didn't have an account so decided to send me feedback here.

The only issues I've run in to so far are both related to the inclination/launch window.  When trying to launch from KSC to the moon the launch window has me launching at a relative inclination of (according to mechjeb) ~8 degrees.  Not huge, but you should be able to make that launch with a relative inclination of ~0.25.  I poked around a bit at the code, but unfortunately my trig is rusty and I have no idea what tricalc is trying to do.  For reference I can see that the c value it is giving me is ~89 degrees.

Launching at an 8 degree relative inclination seems to expose another slightly more straightforward bug.  In closedloopguidance where you update azimuth based on the angle between your orbit and target orbit you're adding the angle to azimuth every time.  When starting with an 8 degree difference it means that every 5 seconds when that loop runs its going to turn 8 degrees more, which in my case left me at about 90 degrees before finally zeroing out the difference, which then had me going off in the other direction by a couple degrees when it gradually swung back at a few degrees every 5 seconds.  My guess is that the math there should be something like azimuth = progradeAzimuth + angle.  My vector math is rusty so I didn't have a chance to test that out

**Comment by QuantumSoundings** [comment]
I thought some of these issues would crop up. I'll look into the launch window problems. The angle correction during flight was kind of a guess really. It works alright if you launch into a relatively close relative inclination, but if the launch window is messed up then it would certainly lead to issues.

**Comment by QuantumSoundings** [comment]
Ok I've tracked down the problem. The way tricalc works is that it relies on knowing that the targets orbit will at some point pass directly overhead of the launch site. It then does trig to calculate some triangle things. The problems comes when the targets orbit does NOT pass directly over the launch site. This only occurs when the inclination of the targets orbit is small than the latitude you are launching from. Such as the moon and cape canaveral,  28.36 and about 28.5 respectively. They way to solve it in this case is to seek to minimize relative inclination. I've implemented that so if you want to try it out I'll be updating the file shortly after some more testing.
